packages/styles/src/elements/bullet-list.scss

```
@@ -0,0 +1,58 @@
+@use '../functions/tokens';
+@use '../tokens/elements';
+
+tokens.$default-map: elements.$post-listbullet;
+
+/* The chars used in suffix are spaces with different widths
+For more information, visit: https://www.fileformat.info/info/unicode/char/search.htm?q=space
+*/
+@counter-style post-bullet {
+  system: extends disc;
+  suffix: '\2007\2009\200a\200a';
+}
+
+ul {
+  list-style: post-bullet;
+  margin-block: tokens.get('list-bullet-margin-block');
+  padding-inline-start: calc(
+    #{tokens.get('list-bullet-item-icon-size')} + #{tokens.get('list-bullet-item-icon-gap-inline')}
+  );
+
+  > li {
+    &::marker {
+      line-height: 1;
+    }
+  }
+
+  &.bullet {
```

Can we prefix the class name with the component name: `.list-bullet`? Could imagine naming conflicts with this class name.

(Would obviously have to be updated throughout the PR).

```suggestion
  &.list-bullet {
```